ERCS/erc-7526.md

```diff
@@ -0,0 +1,128 @@
+This process is referred to as *auto-listing* and is not limited to addresses in the take-back set.
+The token MUST NOT be transferrable for the duration of the auto-listing.
+Implementations MAY specify the duration of the auto-listing window at their discretion but it MUST be finite and SHOULD be long enough to allow arbitrage opportunities to be detected.
+After expiration of the auto-listing, transfer permissions MUST collapse back to standard ERC-721 rules.
+
+In the event of a sale due to an auto-listing:
+
+1. The token MUST be transferred to the address paying the auto-listing price;
+2. The price MUST be transferred to the original token owner;
+3. The corresponding royalties MUST be reimbursed (i.e. the same address receives both the returned royalties and the sale price); and
+4. The new owner MUST be subject to the same mechanism rules; i.e. take-back, royalty payment, and auto-listing.
+
+If no sale occurs during the auto-listing window, the contract SHOULD transfer the royalty payment to the NFT creator.
+
+Marketplaces facilitating sales SHOULD forward royalties in the same transaction as the sale to guarantee payment within the grace period, thus immediately unfreezing the token.
+Marketplaces MAY allow buyers to configure either the auto-listing price or the royalties paid (as the outcome is equivalent), or they MAY provide sensible default values or employ more sophisticated heuristics at their discretion.
+If allowing buyers to configure values, marketplaces MAY limit this to the auto-listing price for ease of understanding as royalties are then conceptually equivalent to ERC-2981's `royaltyInfo(autoListingPrice)`, which is a concept familiar to end users.
+
+```solidity
+/**
+ * @dev This constitutes a rough draft of the minimum required functions, and will almost certainly change as we
+ * complete the reference implementation.
+ */
+interface IERC7526 {
+  function takeBack(uint256 tokenId) external;
+  function payRoyalty(uint256 tokenId) external payable;
+  function buyAutoListing(uint256 tokenId) external payable;
+}
+```
+
+## Rationale
+
+We have published a formal definition of the mechanism and extensive analyses on arXiv[^1].
+
+[^1]:
+    ```csl-json
+    {
+      "type": "article",
+      "id": 1,
+      "author": [
+        {
+          "family": "Huber",
+          "given": "David"
+        },
+        {
+          "family": "Schlosberg",
+          "given": "Arran"
+        }
+      ],
+      "DOI": "10.48550/arXiv.2403.07896",
+      "title": "SACRÉ BLEU: Self-Assessed Creator Royalties Énforced by Balancing Liquidity Estimation & Utility",
+      "original-date": {
+        "date-parts": [
+          [2024, 2, 19]
+        ]
+      },
+      "URL": "https://arxiv.org/abs/2403.07896",
+      "custom": {
+        "additional-urls": [
+        ]
+      }
+    }
+    ```
```

Comment:
If you hold the copyrights to the paper, you can instead include it in the `assets/` directory.